internal/pkg/mapper/defaultbucket/local_load.go

```diff
@@ -0,0 +1,59 @@
+package defaultbucket
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/filesystem"
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/orderedmap"
+)
+
+// MapAfterLocalLoad - replace placeholders with default buckets in IM.
+func (m *defaultBucketMapper) MapAfterLocalLoad(recipe *model.LocalLoadRecipe) error {
+	config, ok := recipe.Object.(*model.Config)
```

Comment:
Nema to byt aj pre config row? Myslim, ze sme to uz riesili, a ze aj v row moze byt IM, tusim ked mas `writer` s config rows.

Alebo je to inak?

Reply:
jo jo, včera jsme říkali že to pošlu zvlášť (myslím 😃). Už jsem to začal lehce připravovat. Ale můžu to dodat klidně sem.

Reply:
A chápu to dobře že sem může přijít v `recipe.Object` config i config row? Takže bych v obou případech z toho vytáhl `.Content` a ten poslal dál do `m.visitStorageInputTables()`, tam bude mít už v podstatě společný kód.

Reply:
Ano, ... ono existuje aj spolocny interface `ObjectWithContent`, ktory mozes pouzit ako typ, ... ci uz v metode alebo podmienke. Len ten interface nema `BranchKey`, takze tu sa asi neda pouzit.

Takze vytiahnut z `config` a `row` `Content` + `BranchKey` a poslat do `visitStorageInputTables`.

Reply:
`včera jsme říkali že to pošlu zvlášť ` moze byt, ... ok, kludne dalsi PR.